#include "base_interfaces\CARLAInterface.h"

int CARLAInterface::getIntValue(std::string base_name) {
	return 0;
};

bool CARLAInterface::getBoolValue(std::string base_name) {
	return true;
};

float CARLAInterface::getFloatValue(std::string base_name) {
	return 0.0;
};

double CARLAInterface::getDoubleValue(std::string base_name) {
	return 0.0;
};

std::string CARLAInterface::getStringValue(std::string base_name) {
	return "";
};

int CARLAInterface::setIntValue(std::string base_name, int value) {
	return 0;
};

int CARLAInterface::setBoolValue(std::string base_name, bool value) {
	return 0;
};

int CARLAInterface::setFloatValue(std::string base_name, float value) {
	return 0;
};

int CARLAInterface::setDoubleValue(std::string base_name, double value) {
	return 0;
};

int CARLAInterface::setStringValue(std::string base_name, std::string value) {
	return 0;
};
